presto-docs/src/main/sphinx/release/release-0.146.rst

```diff
@@ -0,0 +1,9 @@
+=============
+Release 0.146
+=============
+
+* Fix error in map_concat when the second map is empty.
+* Add support for ``VARCHAR(n)`` to Blackhole connector.
```

Comment:
It makes it easier to read. Plus, there might only be one change now, but there could be another release note added, then it has to move. Better to always follow the rule of separate sections per connector.